ayon_api/server_api.py

```diff
@@ -1004,6 +1103,10 @@ def _logout(self):
         logout_from_server(self._base_url, self._access_token)
 
     def _do_rest_request(self, function, url, **kwargs):
+        kwargs.setdefault("timeout", self.timeout)
+        max_retries = kwargs.get("max_retries", self.max_retries)
```

Comment:
Not sure if we should be checking if the passed argument is indeed an `int`, since this would work here but fail later:
```py
kwargs = {"max_retries": ""}```